integrationTests/multiShard/block/interceptedBlocks_test.go

```diff
@@ -174,7 +174,7 @@ func TestMetaHeadersAreRequstedOnlyFromMetachain(t *testing.T) {
 	}
 
 	fmt.Println("Delaying for nodes p2p bootstrap...")
-	time.Sleep(stepDelay)
+	time.Sleep(time.Second * 5)
```

Comment:
is this resilient enough so we won't have to change it back and forth? I don't have a better solution now. Maybe we can have a pattern of `sleepDuration * len(nodes)` if bootstraping more nodes takes more time

Reply:
It is empirical. I have suggested @BeniaminDrasovean in his next PR (integrationTests refactoring) to introduce an exported variable called DelayP2PBroadcast